packages/block-library/src/navigation/edit.js

```diff
@@ -108,7 +112,11 @@ function Navigation( {
 
 	const blockClassNames = classnames( 'wp-block-navigation', {
 		[ `items-justification-${ attributes.itemsJustification }` ]: attributes.itemsJustification,
+		[ fontSize.class ]: fontSize.class,
 	} );
+	const blockInlineStyles = {
```

Comment:
It feels wrong to have to do the inline styles all over the place. Is this a normal practice for font sizes in blocks?

Reply:
Inline styles are used for custom values that don't use predefined classes, so there's really no way around them.
They are also used for custom colors, etc.
Whenever there's a custom style attribute, we just have to use an inline style. 🙂